app/views/devise/sessions/otp_attempt.html.haml

```diff
@@ -0,0 +1,28 @@
+= content_for :page_title_prefix, "Log in"
+
+= form_for(resource, as: resource_name, url: session_path(resource_name), method: :post) do |f|
+  .govuk-grid-row
+    .govuk-grid-column-one-third
+      .govuk-form-group
+        %fieldset.govuk-fieldset
+          %legend.govuk-fieldset__legend.govuk-fieldset__legend--l
+            %h1.govuk-fieldset__heading
+              Verify your login
+
+          = f.govuk_text_field :otp_attempt,
+                                label: { text: "Please enter your six-digit verification code" },
+                                inputmode: "numeric",
+                                pattern: "[0-9]*",
+                                autocomplete: "one-time-code"
+
+  - unless resource.mobile_number_confirmed_at
```

Comment:
worlds smallest thing, but could this be `resource.mobile_number_confirmed_at?` or `resource.mobile_number_confirmed_at.present?`

Reply:
Can add `present?` if you like but this is a DateTime column and can only be `nil` or an `ActiveSupport::TimeWithZone`. `#present?` is really more of use for strings where the blank case is also covered, though I do get that people like it as a visual treat 😄

Reply:
Yup, I understand that, but I like being really explicit where I can be